packages/vite/src/node/optimizer/index.ts

```diff
@@ -31,7 +32,13 @@ const isDebugEnabled = _debug('vite:deps').enabled
 const jsExtensionRE = /\.js$/i
 const jsMapExtensionRE = /\.js\.map$/i
 
-export type ExportsData = ReturnType<typeof parse> & {
+export type { EsModuleLexerImportSpecifier }
+export type EsModuleLexerParseReturnType = readonly [
+  imports: ReadonlyArray<EsModuleLexerImportSpecifier>,
+  exports: ReadonlyArray<string>,
+  facade: boolean
+]
+export type ExportsData = EsModuleLexerParseReturnType & {
```

Comment:
These needs to be exported because it is included in the return type of `vite.optimize`.
But are these needs to be exported? (I didnt check closely yet)

Reply:
We could also export them to be safe, and then we remove them in another PR if they are unneeded.

Reply:
I'll check it and create a new PR if I felt it won't be needed 👍